jetty-util/src/main/java/org/eclipse/jetty/util/IncludeExcludeSet.java

```diff
@@ -233,34 +228,4 @@ public boolean isEmpty()
     {
         return _includes.isEmpty() && _excludes.isEmpty();
     }
-
-    /**
-     * Match items in combined IncludeExcludeSets.
-     * @param item1 The item to match against set1
-     * @param set1 A IncludeExcludeSet to match item1 against
-     * @param item2 The item to match against set2
-     * @param set2 A IncludeExcludeSet to match item2 against
-     * @param <T1> The type of item1
-     * @param <T2> The type of item2
-     * @return True IFF <ul>
-     *     <li>Neither item is excluded from their respective sets</li>
-     *     <li>Both sets have no includes OR at least one of the items is included in its respective set</li>
-     * </ul>
-     */
-    public static <T1,T2> boolean matchCombined(T1 item1, IncludeExcludeSet<?,T1> set1, T2 item2, IncludeExcludeSet<?,T2> set2)
```

Comment:
yay!  As much as I like the logic here, I think that implementing a new Set is much simpler than combining `IncludeExcludeSet`s